### ISSUE TITLE
Please, implement ping from client to Crossbar

### ISSUE THREAD

**Issue by dmugtasimov** [noisyboiler/wampy#72]

**Comment by dmugtasimov** [comment]
No, it is not a bug report, rather a feature request. Now it is implemented to pong a ping from Crossbar. I would like also to be able to send ping from client to crossbar.

**Comment by noisyboiler** [comment]
Ok. So would a configurable websocket Ping interval be what you are after? And whatever `wampy` client you instantiate would Ping the router at these intervals?

**Comment by dmugtasimov** [comment]
I see it like I have method on `wampy` client which I call whenever I want and it sends ping and I have a decorator to mark a handler for pong. I this case it is up to `wampy` user to ping in intervals or on demand. Also pong timeout is up to `wampy` user to validate in handler (or outside of it).

**Comment by noisyboiler** [comment]
are you being disconnected from a Router then? if so, `wampy` does return a `Pong` if a Router pings you... maybe it just needs to be configured to do so?

Or if you're not being disconnected, I'm curious as to why you'd like this.

We could do it, but it needs some thought. `wampy` only has public APIs to send `WAMP` messages (no big surprise there) and exposing the websocket messages to the user is going to break the design.

**Comment by noisyboiler** [comment]
@dmugtasimov 
i gave it a go tonight: https://github.com/noisyboiler/wampy/pull/86/files
